index.html

```diff
@@ -4603,6 +4603,11 @@ <h3>Interactability</h3>
  <li><p>Return the pair of (<var>x</var>, <var>y</var>).
 </ol>
 
+<p>An <a>element</a> is <dfn>disabled</dfn> when it is an <a>element</a>
```

Comment:
That's not quite what your gecko patch does; it looks for any disabled `<select>` or `<optgroup>` ancestor, not just the nearest `<select>` ancestor. The gecko patch seems correct to me, since it makes the `<select disabled><optgroup><option>` and `<select><optgroup disabled><option>` cases work.

I'd be somewhat tempted to write this as an algorithm, like

```
An element |element| is <dfn>disabled</dfn> if the following steps return true:

1. If |element| is an <{option}> element or |element| is an <{optgroup}> element:

  1. For each [=inclusive ancestor=] |ancestor| of |element|:

    1. If |ancestor| is an <{optgroup}> element or |ancestor| is a <{select}> element, and |ancestor| is [=actually disabled=], return true.

  1. Return false.

1. Return|element| is [=actually disabled=].
```

Reply:
Oh, I actually forgot to update the PR here after applying the changes to the Gecko patch. So yes the `optgroup` needs to be added here as well.

Your suggestion looks fine but would require some small updates for the references. Does it look fine to you now? Whom else do we have to ask for review?